### PR TITLE
Update pro feature module and add signout link in header

### PR DESCRIPTION
**Pull request by whmii**
screen shot 2015-03-03 at 2 41 28 pm
screen shot 2015-03-03 at 11 01 05 am

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -31,9 +31,16 @@
...
-          <%= link_to "Sign in", sign_in_path, class: "app-header-sign-in" %>
-        <% end %>
+        <nav class="app-header-nav">
+          <% if signed_out? %>
+            <%= link_to "Sign in", sign_in_path, class: "app-header-sign-in" %>
+          <% else %>
+            <span class="app-header-username">
+              <%= current_user.email %>
+            </span>
+            <%= link_to "Sign out", sign_out_path, class: "app-header-sign-in" %>
```
Comment (ckundo) [inline app/views/layouts/application.html.erb]: Should `app-header-sign-in` have a more generic name?

File: app/views/shared/_accesspro.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<section class="accesspro" role="contentinfo">
+  <header class="accesspro-header">
+    <%= render 'shared/logo_betabadge.svg' %>
+    <h2 class="accesspro-title">Beta</h2>
```
Comment (ckundo) [inline app/views/shared/_accesspro.html.erb]: Is this really a heading?

File: config/routes.rb [hunk 3]
```diff
@@ -11,7 +11,7 @@
   end
 
   get "/sign_in" => "clearance/sessions#new", as: "sign_in"
-  delete "/sign_out" => "clearance/sessions#destroy", as: "sign_out"
+  get "/sign_out" => "clearance/sessions#destroy", as: "sign_out"
```
Comment (ckundo) [inline config/routes.rb]: What does clearance docs say about this? I think the delete action could be more appropriate, but whatever the docs indicate should do.

**Comment by ckundo** [comment]
See what you think of the notes. The main thing is the clearance route question, but if that's kosher then :rocket:

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (whmii) [inline app/views/layouts/application.html.erb]: Actually it's no longer necessary so… :scissors:

File: app/views/shared/_accesspro.html.erb (hunk 2 above)
Comment (whmii) [inline app/views/shared/_accesspro.html.erb]: well it is a-head of the rest! amirite!?

File: config/routes.rb (hunk 3 above)
Comment (whmii) [inline config/routes.rb]: Delete wouldn't function and would just throw an error so i replaced it with `get`.  This was just me monkey patching but if there is a real way to do this i'm all for it.

japbcvpavbzau9dbuaxf-1

File: app/assets/stylesheets/_app-header.scss [hunk 4]
```diff
@@ -17,11 +17,20 @@
...
   }
 
-  .app-header-sign-in {
+  .app-header-nav {
     float: right;
     line-height: 4.25;
     text-decoration: none;
     white-space: nowrap;
+
+    a {
```
Comment (whmii) [inline app/assets/stylesheets/_app-header.scss]: Appropriate in this context

File: app/assets/stylesheets/base/_buttons.scss [hunk 5]
```diff
@@ -1,11 +1,45 @@
...
+    box-shadow: $form-box-shadow-focus;
+    outline-color: $base-accent-color;
+  }
+
+  &-large {
+    padding: 0.75em 2em;
+  }
+}
+
+input[type="submit"] {
```
Comment (whmii) [inline app/assets/stylesheets/base/_buttons.scss]: necessary for baseline overides.

File: config/routes.rb (hunk 3 above)
Comment (ckundo) [inline config/routes.rb]: `<%= button_to "Sign out", sign_out_path, method: :delete %>`

from https://github.com/thoughtbot/clearance/blob/master/README.md#helper-methods :+1:

Comment (whmii) [inline config/routes.rb]: Works like a charm :heart:

**Pull request merged**